### ISSUE TITLE
[regression] 252.4: "systemd-udevd: event_source: Failed to get device name: No such file or directory"

### ISSUE THREAD

**Issue by enihcam** [systemd/systemd#25811]
### systemd version the issue has been seen with

252.4

### Used distribution

Archlinux

### Linux kernel version used

5.15.84-xanmod1-1-lts

### CPU architectures issue was seen on

x86_64

### Component

systemd-udevd

### Expected behaviour you didn't see

no such error-level messages like `systemd-udevd: event_source: Failed to get device name: No such file or directory`

### Unexpected behaviour you saw

`systemd-udevd: event_source: Failed to get device name: No such file or directory` is treated as an error.

### Steps to reproduce the problem

upgrade systemd from 252.3 to 252.4

### Additional program output to the terminal or log subsystem illustrating the issue

```
Dec 21 15:12:16 systemd-udevd[202]: event_source: Processing device (SEQNUM=761, ACTION=add)
Dec 21 15:12:16 systemd-udevd[202]: event_source: /usr/lib/udev/rules.d/60-evdev.rules:9 Importing properties from results of builtin command 'hwdb --subsystem=input --lookup-pref>
Dec 21 15:12:16 systemd-udevd[202]: event_source: No entry found from hwdb.
Dec 21 15:12:16 systemd-udevd[202]: event_source: /usr/lib/udev/rules.d/60-evdev.rules:9 Failed to run builtin 'hwdb --subsystem=input --lookup-prefix=evdev:': No data available
Dec 21 15:12:16 systemd-udevd[202]: event_source: /usr/lib/udev/rules.d/60-evdev.rules:28 Importing properties from results of builtin command 'keyboard'
Dec 21 15:12:16 systemd-udevd[202]: event_source: /usr/lib/udev/rules.d/60-evdev.rules:28 Failed to run builtin 'keyboard': No such file or directory
Dec 21 15:12:16 systemd-udevd[202]: event_source: Device processed (SEQNUM=761, ACTION=add)
Dec 21 15:12:16 systemd-udevd[202]: event_source: sd-device-monitor(worker): Passed 187 byte to netlink monitor.
Dec 21 15:12:16 systemd-udevd[202]: event_source: Failed to get device name: No such file or directory
```

**Comment by enihcam** [comment]
it is probably introduced by  https://github.com/systemd/systemd/commit/953c928c24455744d5534679998d129b947a5e04

@keszybz

**Comment by yuwata** [comment]
I will take a look.

**Comment by yuwata** [comment]
Fix is waiting in #25812.
